doc/DeveloperGuide.md

```diff
@@ -94,7 +94,12 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
-
+5. The system should be able to run in 32-bit and 64-bit systems.
+6. The system should be intuitive and user-friendly.
+7. The system should be stable with minimal crashes.
```

Comment:
Think about how you can make this precise. How minimal is minimal? What do you mean by stable?

Reply:
By minimal crashes, maybe it would be better for me to quantify the number of crashes in a set amount of time?